src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -9,7 +16,21 @@ void EnSyatekiNiw_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnSyatekiNiw_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnSyatekiNiw_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80B11DEC(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B132A8(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B131B8(EnSyatekiNiw* this, Vec3f* arg1, Vec3f* arg2, Vec3f* arg3, f32 arg4);
+void func_80B129EC(EnSyatekiNiw* this, GlobalContext* globalCtx);
```

Comment:
id group all of the action funcs together and then put any extra stuff under it with a line break